generators/app/templates/src/components/Routes/constants.tsx

```diff
@@ -5,28 +5,27 @@ import { User } from 'contexts/UserContext/reducer';
 
 import PATHS from './paths';
 
-const Home = lazy(() => import('../../screens/Dashboard'));
+const Home = lazy(() => import('../../screens/Dashboard/screens/Home'));
+const Logout = lazy(() => import('../../screens/Dashboard/screens/Logout'));
 // Add imports for screens above (FOR GENERATORS, DO NOT REMOVE)
 
-const MAIN_PUBLIC_PATH = PATHS.login;
+const MAIN_PUBLIC_PATH = PATHS.logout;
 const MAIN_PRIVATE_PATH = PATHS.home;
 
 /* When adding routes, add them ABOVE the Home route
  * or it will redirect you to incorrect screens */
 export const ROUTES = [
   // Leaving this as an example for then the Login screen exists
   {
-    exact: false,
-    path: PATHS.login,
-    component: () => <span>login</span>,
+    path: PATHS.logout,
+    element: <Logout />,
```

Comment:
I think you should revert this to `Login`

Reply:
The element property now receives a component and not a string